CHANGELOG.md

```diff
@@ -1,6 +1,65 @@
 # go-ipfs changelog
 
-## 0.4.16
+## 0.4.17-rc1 2018-07-20
+
+Ipfs 0.4.17 is a quick release to fix two important bugs:
+
+1. A major performance regression in bitswap (mostly affecting go-ipfs ->
+   js-ipfs transfers).
+2. A bug resolving files in sharded directories with the gateway.
```

Comment:
TBH that issue wasn't present in 0.4.16 - was introduced in https://github.com/ipfs/go-ipfs/pull/4672 which got merged after that release

Reply:
Hm. Good point. We should fix that before the actual release.